CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ See updating [Changelog example here](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+### Added
+- Managed Load Balancer private network support
+
+### Deprecated
+- Managed Load Balancer fields `DNSName` and `network_uuid`
```

Comment:
Could use go field name instead of JSON field name also for `NetworkUUID`

Reply:
Yep should be `NetworkUUID`. I'll push update.